packages/outstatic/package.json

```diff
@@ -33,6 +33,8 @@
     "@tiptap/react": "2.0.0-beta.202",
     "@tiptap/starter-kit": "2.0.0-beta.202",
     "@uiw/react-md-editor": "3.6.0",
+    "camelcase": "6.3.0",
+    "capitalize": "^2.0.4",
```

Comment:
If we're going to force-capitalize items such as the accordion title, we may want to add a corresponding `title` attribute that tells them the content location like `/posts`